projects/verifiable-credential-wallet.md

```diff
@@ -0,0 +1,59 @@
+# Project Name
+Verifiable Credential Wallet
+
+# Preferred Maturity Level
+Labs stage
+
+# Project Description
+The Verifiable Credential Wallet is a cross-platform iOS and Android mobile application for storing and sharing digital credentials, starting with higher learning.
+
+The wallet is based on the learner credential wallet specification developed by the Digital Credentials Consortium. 
+
+The app works with iOS and Android and allows users to add and share credentials, as well as manage the wallet.
+
+Whenver possible, we use existing open standards from W3C, IETF, and other open standards groups.
+
+
+# Alignment with the OpenWallet Foundation Mission
+This project aligns with the OWF mission, because it's an open source example implementation of wallet-related open standards.
+
+# Code of Conduct
+https://wiki.dcconsortium.org/app/page/1SYYIdVWKAdBIBx3BwSFWZKytJ9zPprAZFTbD_7Bfo4Y?p=1h1VJHHv2zSe0n9Ltg-KQevSuqWbGcExy
+
+# TAC Sponsor
+
+
+# Project License
+MIT License but can change to Apache 2.0 and will leave open to discussion.
+
+# Source Control
+https://github.com/digitalcredentials/learner-credential-wallet
+
+# Issue Tracker
+https://github.com/digitalcredentials/learner-credential-wallet/issues
+
+# External Dependencies
+See package.json (https://github.com/digitalcredentials/learner-credential-wallet/blob/main/package.json) - Have ensured that all dependencies are all open source.
```

Comment:
Did we check for permissive and non permissive licenses?

Reply:
@skounis We've checked that all dependencies are open source, which would include permissive/non permissive.